patches/net/minecraft/data/DataProvider.java.patch

```diff
@@ -1,9 +1,11 @@
 --- a/net/minecraft/data/DataProvider.java
 +++ b/net/minecraft/data/DataProvider.java
-@@ -27,7 +_,15 @@
+@@ -26,8 +_,16 @@
+ import net.minecraft.util.GsonHelper;
  import org.slf4j.Logger;
  
- public interface DataProvider {
+-public interface DataProvider {
```

Comment:
Oh I my assumption was wrong, I remembered seeing this PR (#1315) and thought our condition system supported any data file now, but guess I was wrong as it was closed not merged.

I can go through and find all the providers which directly support conditions, rather than implementing onto `DataProvider`.